src/components/subscriptionDialog.vue

```diff
@@ -0,0 +1,286 @@
+<template>
+  <v-dialog
+    id="subscription-dialog"
+    v-model="showSubscriptionDialog"
+    max-width="550"
+  >
+    <v-card style="display: flex; flex-direction: column; overflow: hidden;">
+      <div id="subscription-details">
+        <v-card-title class="headline">
+          Premium Subscription
+        </v-card-title>
+        <v-card-text data-cy="detail">
+          Subscribe to Ambianic Premium Services to enable advanced
+          features, such as instant email notification.
+        </v-card-text>
+        <div>
+          <h3
+            data-cy="price"
+            style="font-weight: 500; text-align: center;"
+          >
+            $5 Monthly Fee
+          </h3>
+          <br>
+        </div>
+      </div>
+
+      <div
+        id="error"
+        v-if="subscriptionError"
+      >
+        <p class="error-text">
+          An error occurred while adding your subscription. Please try again
```

Comment:
Yes. 

There is a plan to include more detailed error logging in the `Settings` page of the Ambianic PWA. This includes the redesigning of the Settings page and is being documented and tracked in this issue, however that issue is dependent on this.

I would improve the error message in the `SubscriptionDialog` component based on your recommendation. 

As discussed on Slack, integrating an external error monitoring tool like [Sentry](https://sentry.io) is being considered, however, it may be a long-term objective.

Reply:
@vickywane  OK, please open an issue to track external cloud API monitoring.